crates/solver/src/solver/http_solver.rs

```diff
@@ -175,8 +175,12 @@ fn map_tokens_for_solver(orders: &[LimitOrder], liquidity: &[Liquidity]) -> Vec<
 }
 
 fn order_fee(order: &LimitOrder) -> FeeModel {
+    let amount = match order.is_liquidity_order {
+        true => U256::zero(),
```

Comment:
Instead of 0, should this be the actual fee?
```suggestion
        true => order.actual_fee, // field doesn't currently exist, but maybe we should add it?
```

Reply:
Thanks Nick!
Yes, I was under the impression that all liquidity orders would be charged a fee of zero, hence the shortcut. But you are right that the actual amount is the right spot.


Here, a quick explainer why liquidity orders were included before, but now shouldn't be anymore:

Normally, liquidity orders would not be included as the overall objective, as long as the following would hold:
```
 fees of liqudity order - cost + surplus 
 ```
 But before this PR, we had:
 ```
 fees of liqudity order  + fee subsidy - cost + surplus 
 ```
 Now, it happened that fee subsidy was sometimes higher than the cost. Then these orders would be able to steal surplus from other orders as long as their `fees of liqudity order ` was higher than the surplus stolen from another order

Reply:
I will add the additions to the overall objective function from the liquiidty orders (their new fee) into the function `total_scaled_unsubsidized_fees` in another PR

Reply:
Created #1763 to not forget.

Reply:
Don't we also need to change the solvers themselves?

Reply:
Nevermind, I think the purpose of this PR is to make the driver as it is already in the solver, right?